.devcontainer/db/Dockerfile

```
@@ -8,8 +8,7 @@ LABEL description="OpenOSP-EMR Docker Image for Development Environment"
 # Install required packages
 RUN apt-get update && apt-get install -y dos2unix  \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)